CARTOGRAPHY.md

```diff
@@ -37,3 +37,51 @@ account.
 * **A rich map** - This style deliberately creating a fairly rich map showing a significant number of different features.  This way it shows the richness of OSM data and gives a broad recognition to the mappers' work.  The aim is not however to show all or even most of the OSM data.
 * **Maintainability** - The implementation of this style should not be too hard to maintain. This refers to both the volume and complexity of the code and how fast the style can be parsed when rendering it, which is very important for efficient development work. So the amount of code should be kept small and complex and fragile interdependencies should be avoided. If the code is difficult to maintain this would ultimately seriously affect all of the above goals.
 * **Adaptability and ease of use** - The style should be easy to customize, like for creating localized or special purpose maps. It is also important to keep demands on rendering infrastructure for serving the style low so it is not too difficult and costly to set up a tile server for this style or a specialized variant of it.
+
+## Design guidelines
+
+The following guidelines have been found to be useful design principles that can help creating a better map in line with the above goals.  Keep in mind that none of these principles represents a goal on its own and most of them are highly condensed summaries of more complex considerations.
+
+* The difference in rendering between different types of features should foremost be based on their difference in meaning and purpose for the target map users.  Differences in physical appearance are only relevant as far as they also represent differences in meaning and purpose.
+
+* Design of symbols, area and line patterns should be in a way that avoids their geometries to be mistaken as actual data.
+
+* The connection between data and appearance of features in the map needs to be comprehensible for the viewer.
+
+* When changes have a strong impact on the overall appearance of the map input should be sought from the broader OpenStreetMap community and particular consideration given to their comments and concerns.
+
+### Colors
+
+* Differentiating features through different colors only makes sense if this difference can actually be read by the map users.
+
+* For area colors we prefer light and low saturation tones.  Lines, individual symbols and labels also can be stronger in color.
+
+* Stronger colors indicate fairly distinct and meaningful features as opposed to more general and less distinct features which use weaker colors.
+
+* Small areas work well with somewhat stronger colors while large areas tend to call for weaker tones.
```

Comment:
What is important for me is if it works for this style. If we know the typical size (which is not always true) this is not always "smaller is more important". On the other hand I would also not argue that "bigger is more important", that's why I think it's hard to say about a rule here.

1. When working with midzoom I see that the area size doesn't matter for colors - it's rather areas vs lines and points (and you have proposed such rule in this PR, which I agree with).

2. When working with pitches I see that they are typically a part of the school area or sport centres. Both of these are now rendered as pale yellow and dark pitches/tracks were hiding the fact that they are just a part of the feature. We loose the feeling of having the structure, which I consider to be a bad thing.

3. The same with park. I see it like no matter if we have tree area or the grass area or whatever else, we're still basically in the park, this is the important frame for all the features inside and it [looks bad](http://www.openstreetmap.org/relation/3921018) when smaller areas make it not clear.

4. The land color is light as a bigger area than most of the others, and here is where this rule works for me. But if you can find both valid examples and counterexamples, we can't really talk about the rule any more.